### ISSUE TITLE
LineageOS?

### ISSUE THREAD

**Issue by rugk** [mar-v-in/freecyngn#42]
Did you include the changes into LineageOS?
Or - in general - is LineageOS more open than CM?

**Comment by ghost** [comment]
From #40 and #36, the non-free component left in LineageOS are

- Gello
- Unrar
- TimeService

More info regarding LineageOS in those threads.

**Comment by ghost** [comment]
@nvllsvm @mar-v-in what about a new release for CM14/LineageOS?

**Comment by andrewrabert** [comment]
@kehugter It will be updated in a day or two. :)

**Comment by andrewrabert** [comment]
**Gello**: It has been [removed from LineageOS](https://lineageos.org/Changelog-9/) in favor of Jello.
**unrar**: The full source code is available for auditing. While the license is not ideal, we can at least guarantee it's not violating our privacy. I don't see the point in removing this if we are keeping proprietary firmware and drivers.
**TimeService**: Not clear on what this does. Since it seems related to firmware, I'd be hesitant to remove it.

**tl;dr** - I see no reason for this project to exist anymore.

**Comment by CryptGoat** [comment]
Pretty much agreed. Still, thanks for your work on this! =) 

You might want to provide some info about the status of the project and the reasons for stopping development on the project page.

**Comment by ghost** [comment]
I was flashing mainly to avoid `CyanogenSetupWizard` pimping for a google account. Is that part of the setup gone too?
